src/App.jsx

```diff
@@ -2,17 +2,16 @@ import { useState } from 'react';
 import './App.scss';
 
 export const App = () => {
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
 
   const addOne = () => {
-    // write code here
+    setCount(cCount => cCount + 1);
```

Comment:
```suggestion
    setCount(prevCount => prevCount + 1);
```
make more meaningful name for previous state.
also fix in add100 funcion